src/commands/deploy/metadata/preview.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
```

Comment:
2022

Reply:
I wonder why the linter doesn't fix these :shrug: